# SYSTEMS ANALYSIS OF TRANSCRIPTOMIC CROSSTALK IN KNEE JOINT FOLLOWING LOAD-INDUCED OSTEOARTHRITIS AND TREATMENT IN MICE 
![image](https://github.com/Adrien1995/Spatiotemporal-network-analysis-identifies-gene-clusters-driving-PTH-ALN-treatment-effects-during-lo/assets/60327600/f48eafde-207b-4275-8614-655c0cdf64bf)

Abstract

The knee joint is a whole-organ system yet isolated tissues are analyzed independently when characterizing gene expression associated with pathologies. This approach is particularly limiting for diseases such as osteoarthritis (OA) that affects multiple tissues and their crosstalk. Here we used a systems biology approach to integrate information from multiple tissue transcriptomes and better characterize joint tissue crosstalk during disease progression. Our objective was to understand the multi-directional transcriptomic crosstalk shared among tissues during OA development and the effects of treatments. We subjected mice to daily tibial loading to initiate OA-like pathology and study load magnitude- and time-dependent changes in the knee joint. Leveraging gene co-expression networks, we identified an immune-related network of genes conserved across tibial cartilage, metaphyseal cortical bone, cancellous bone, and bone marrow, and inguinal lymph nodes that was upregulated with load-induced OA. Among the top five hub genes, two genes previously associated with OA were identified: Tbc1d10c, a cytoskeletal functional linker known to modulate cell spreading and phagocytosis in macrophages and recently discovered within loci associated with OA and, Rab4b, found to be upregulated in synovitis of human OA. Additionally, using this system approach, we investigated potential drug candidates and shortlisted three potential therapeutic targets for further investigation in load-induced OA. Overall, considering the combined gene network interactions provided a more holistic understanding of the multi-tissue effects of treatment following cyclic tibial compression.



Structure

1.code/ contains the code for cleaning, adjusting and normalizing individual datasets and performing the cross-tissue transcriptome analyses

2.raw_data/ contains individual-level study data for RNAseq and single-cell RNAseq

3.results/ contains figures and table results from the cross-tissue analyses

4.working_data/ contains intermediary files

*available upone publication in peer-review journal
